tests/test_datasetattributes.py

```diff
@@ -285,4 +377,79 @@ def test_assign_labels_to_points(hexbeam):
     hexbeam.point_arrays.clear()
     labels = [f"Label {i}" for i in range(hexbeam.n_points)]
     hexbeam['labels'] = labels
-    assert np.all(hexbeam.active_scalars == labels)
+    assert (hexbeam['labels'] == np.array(labels)).all()
```

Comment:
We probably don't need the array conversion on the right, but I haven't tested this
```suggestion
    assert (hexbeam['labels'] == labels).all()
```

Reply:
Tested locally, this works.